packages/remix-dev/compiler/plugins/browserRouteModulesPlugin.ts

```diff
@@ -38,7 +38,11 @@ export function browserRouteModulesPlugin(
       );
 
       build.onResolve({ filter: suffixMatcher }, args => {
-        return { path: args.path, namespace: "browser-route-module" };
+        return {
+          path: args.path,
+          namespace: "browser-route-module",
+          sideEffects: false
```

Comment:
Why do we need this?

Reply:
We don't, leftover code that I removed :)